Assets/Scripts/InteractionSystem/InteractableObjects/NPCInteractionBehavior.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using PlaceboEntertainment.UI;
+
+public class NPCInteractionBehavior : MonoBehaviour, IInteractable
+{
+    [SerializeField] private string _npcName;
+    public void Interact(GameObject player)
+    {
+        GetComponent<BaseNpc>().Interact();
+    }
+
+    public void DisplayInteractUI()
+    {
+        TabbedMenu.Instance.ToggleInteractPrompt(true, _npcName);
```

Comment:
Instead of displaying the NPC name, would it not make more sense to just display "TALK"? Should ask designers for their input.